hoverxref/extension.py

```diff
@@ -337,6 +337,13 @@ def setup_theme(app, exception):
         )
 
 
+def setup_assets_policy(app, exception):
+    """Tell Sphinx to always include assets in all HTML pages."""
+    if sphinx.version_info >= (4, 1, 0):
```

Comment:
Should we do duck-typing here, `if hasattr(app, 'set_html_assets_policy')`, to make this more resilient?

Reply:
I don't have a strong opinion about this. I've been using `version_info` for other similar cases, so I just followed that here. It also works as a way to auto-document when it changed, but... No other reasons, honestly.

Reply:
I'd much prefer not relying on the Sphinx version. That also unblocks this release, and it will just start working with any compatible version.

Reply:
We can just put a comment saying `# Introduced in Sphinx 4.1` or something, but makes the code nicer.

Reply:
Updated!